package.json

```diff
@@ -10,7 +10,8 @@
   "author": "Max Brunsfeld",
   "license": "MIT",
   "dependencies": {
-    "nan": "^2.12.1"
+    "nan": "^2.12.1",
+    "tree-sitter": "^0.16.1"
```

Comment:
I don't know why this was added or why it was not here before. I called `npm install tree-sitter` and `npm install tree-sitter-cli` beforehand.

Reply:
IIRC `tree-sitter` is the NodeJS bindings. Shouldn't necessary for writing a grammar. It's there because part of `npm install <pack>` is to add `<pack>` to dependencies. Just run `npm install` to install all the existing package dependencies. I believe `tree-sitter-cli` is the package that provides the `tree-sitter` CLI program.

Reply:
I see Travis uses just `npm install` and `npm test` and this doesn't affect `package.json`.